tests/ui/coroutine/clone-impl.stderr

```diff
@@ -117,11 +126,14 @@ LL |     move || {
 LL |     check_clone(&gen_non_clone);
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^ within `{coroutine@$DIR/clone-impl.rs:79:5: 79:12}`, the trait `Clone` is not implemented for `NonClone`, which is required by `{coroutine@$DIR/clone-impl.rs:79:5: 79:12}: Clone`
    |
-note: captured value does not implement `Clone`
-  --> $DIR/clone-impl.rs:81:14
+note: coroutine does not implement `Clone` as this value is used across a yield
```

Comment:
Why does this change?

Reply:
Because the diagnostic has not been updated to recognise the locals or fields in `Unresumed` variant as captured values. Let me work on that!

Reply:
OK, as long as it's not a change in behavior (like I mentioned above w/ that other example where the test changed) this could either be done here or in a follow-up.